app/views/admin/campaigns/_campaigns.html.erb

```diff
@@ -0,0 +1,31 @@
+<%= hidden_field_tag :direction, params[:direction] %>
+<%= hidden_field_tag :sort, params[:sort] %>
+<table class="table">
+  <thead>
+    <tr>
+      <th class="sortable"><%= sortable "name" %></th>
+      <th>Description</th>
+      <th>Htag</th>
+      <th class="sortable"><%= sortable "start_date" %></th>
+      <th class="sortable"><%= sortable "status" %></th>
+      <th>Actions</th>
+    </tr>
+  </thead>
+  <tbody>
+    <% for campaign in @campaigns %>
+      <tr>
+        <td class="-bold"><%= campaign.name %></td>
+        <td><%= campaign.description %></td>
+        <td><%= campaign.htag %></td>
+        <td><%= campaign.start_date %></td>
+        <td><%= campaign.status_humanize %></td>
+        <td class="actions">
+          <%= link_to '', campaign_path(campaign), :class => "svg-bttn -show" %>
```

Comment:
As we are linking to the public interface here, and there's no easy way to come back here, I'd open this and other show links in a new tab. Unless your goal was to link to the show page of the admin, then you'll need to update the path you are using.